src/platform/backends/qemu/qemu_virtual_machine.h

```diff
@@ -22,9 +22,9 @@
 #include <multipass/optional.h>
 #include <multipass/virtual_machine.h>
 
+#include <QStringList>
+
 class QProcess;
-class QFile;
-class QString;
```

Comment:
This some drive-by cleanup?

Reply:
Yes, as I mentioned earlier, I like focused PR's, but I knowingly did this here since it's quite benign.  But if you prefer, I can also revert this and propose in another PR.

Reply:
As mentioned on IRC, it's ok to leave this.